hack/lib/util/golang.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#
+# This library holds golang related utility functions.
+
+# os::golang::verify_go_version ensure the go tool exists and is a viable version.
+function os::golang::verify_go_version() {
+  if ! which go &>/dev/null; then
+    os::log::info "Can't find 'go' in PATH, please fix and retry."
```

Comment:
This was my fault in the previous comment, but these should be `os::log::error` since they mean something failed.